apps/docs/docs/reference/ui/atoms/Radio.mdx

```diff
@@ -1,31 +1,173 @@
 ---
 description: Radios allow users to select one option from a set.
-sidebar_custom_props: 
+sidebar_custom_props:
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Radio___4bdff3538eb6a18e99c6f5a9c1043468.png
 ---
 
-import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
 # Radio
 
 Radios allow users to select one option from a set.
 
+## Overview
+
+```tsx live
+<div style={{ display: 'flex', columnGap: '16px' }}>
+  <Radio />
+  <Radio checked readOnly />
+  <Radio disabled />
+  <Radio checked disabled readOnly />
+</div>
+```
+
+<small>
+  You can remove <code>readOnly</code> prop when using.
+</small>
```

Comment:
Is this bit necessary? I think it's confusing

Reply:
Not sure, just in case the person copy the whole code, but maybe we can remove it.